From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: lmsdscav <lmsdscavmc@hotmail.com>
Date: Sun, 4 Feb 2024 15:51:06 +0800
Subject: [PATCH] Phosgene Rebrand server


diff --git a/build.gradle.kts b/build.gradle.kts
index 99cf0bebf6bd1579c1f347e0c1750c7b13f74566..2fd675b842c150475458c99c7d21f6985c3e0d18 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -15,7 +15,7 @@ val alsoShade: Configuration by configurations.creating
 dependencies {
     // Gale start - project setup
     // Depend on own API
-    implementation(project(":leaf-api")) // Leaf
+    implementation(project(":phosgene-api")) // Leaf
     // Depend on Paper MojangAPI
     implementation("io.papermc.paper:paper-mojangapi:1.20.1-R0.1-SNAPSHOT") {
         exclude("io.papermc.paper", "paper-api")
@@ -106,7 +106,7 @@ tasks.jar {
         attributes(
             "Main-Class" to "org.bukkit.craftbukkit.Main",
             "Implementation-Title" to "CraftBukkit",
-            "Implementation-Version" to "git-Leaf-$implementationVersion", // Gale - branding changes // Leaf
+            "Implementation-Version" to "git-Phosgene-$implementationVersion", // Gale - branding changes // Leaf // Phosgene
             "Implementation-Vendor" to date, // Paper
             "Specification-Title" to "Bukkit",
             "Specification-Version" to project.version,
