From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: lmsdscav <lmsdscavmc@hotmail.com>
Date: Sun, 4 Feb 2024 16:15:21 +0800
Subject: [PATCH] Phosgene add basic class


diff --git a/src/main/java/org/dreeam/leaf/LeafConfig.java b/src/main/java/org/dreeam/leaf/LeafConfig.java
index bf4c5ce82ecee2e7040d6cea173df7827606a8b8..8687c9b779f569df90948773b7fd948ac169cbe5 100644
--- a/src/main/java/org/dreeam/leaf/LeafConfig.java
+++ b/src/main/java/org/dreeam/leaf/LeafConfig.java
@@ -12,7 +12,6 @@ import org.jetbrains.annotations.Nullable;
 import org.simpleyaml.configuration.comments.CommentType;
 import org.simpleyaml.configuration.file.YamlFile;
 import org.simpleyaml.exceptions.InvalidConfigurationException;
-
 import java.io.File;
 import java.io.IOException;
 import java.lang.reflect.Method;
diff --git a/src/main/java/org/lxstd/phosgene/Phosgene.java b/src/main/java/org/lxstd/phosgene/Phosgene.java
new file mode 100644
index 0000000000000000000000000000000000000000..adb67e1df48bfe9f88d490e396d4e8044b515500
--- /dev/null
+++ b/src/main/java/org/lxstd/phosgene/Phosgene.java
@@ -0,0 +1,4 @@
+package org.lxstd.phosgene;
+
+public class Phosgene {
+}
diff --git a/src/main/java/org/lxstd/phosgene/phosgeneConfig.java b/src/main/java/org/lxstd/phosgene/phosgeneConfig.java
new file mode 100644
index 0000000000000000000000000000000000000000..0464dad9329ad9ff682b254e8507a1367c952d0d
--- /dev/null
+++ b/src/main/java/org/lxstd/phosgene/phosgeneConfig.java
@@ -0,0 +1,5 @@
+package org.lxstd.phosgene;
+
+public class phosgeneConfig {
+    // no need right now
+}
