From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: lmsdscav <lmsdscavmc@hotmail.com>
Date: Sun, 4 Feb 2024 16:43:08 +0800
Subject: [PATCH] Phosgene ThreadPool CountDownLatch Add


diff --git a/src/main/java/org/lxstd/phosgene/Phosgene.java b/src/main/java/org/lxstd/phosgene/Phosgene.java
index 91457f97871fa05c1fea3f8927e180894a7fde05..d2bd29cb2a1a8298728b45a779398423a860aa12 100644
--- a/src/main/java/org/lxstd/phosgene/Phosgene.java
+++ b/src/main/java/org/lxstd/phosgene/Phosgene.java
@@ -1,5 +1,6 @@
 package org.lxstd.phosgene;
 
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
@@ -8,6 +9,18 @@ public class Phosgene {
     public static void pushTask(Runnable task){
         PhosgeneThreadPool.execute(task);
     }
+    public static void pushTask(Runnable task,CountDownLatch launcher){
+        //PhosgeneThreadPool.execute(task);
+        PhosgeneThreadPool.execute(()->{
+            try{
+                task.run();
+            }catch (Exception e){
+                e.printStackTrace();
+            }finally {
+                launcher.countDown();
+            }
+        });
+    }
 
 
 }
