From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: lmsdscav <lmsdscavmc@hotmail.com>
Date: Sun, 4 Feb 2024 16:54:02 +0800
Subject: [PATCH] Phosgene Rebrand Force can not be changed


diff --git a/src/main/java/org/dreeam/leaf/LeafConfig.java b/src/main/java/org/dreeam/leaf/LeafConfig.java
index 2dc1567c03fd1771281d20e6a5342cd2e6e900ce..9c5aa41cc4844578a3ca62978892f82edfec41d8 100644
--- a/src/main/java/org/dreeam/leaf/LeafConfig.java
+++ b/src/main/java/org/dreeam/leaf/LeafConfig.java
@@ -169,7 +169,8 @@ public class LeafConfig {
 
     public static String serverModName = "Phosgene";
     private static void serverModName() {
-        serverModName = getString("server-mod-name", serverModName);
+        //serverModName = getString("server-mod-name", serverModName);
+        serverModName = "Phosgene";
     }
 
     public static boolean removeMojangUsernameCheck = true;
